addons/website_event_track/models/event_track_stage.py

```diff
@@ -19,14 +19,11 @@ class TrackStage(models.Model):
     fold = fields.Boolean(
         string='Folded in Kanban',
         help='This stage is folded in the kanban view when there are no records in that stage to display.')
-    is_accepted = fields.Boolean(
-        string='Accepted Stage',
-        help='Accepted tracks are displayed in agenda views but not accessible.')
-    is_done = fields.Boolean(
-        string='Done Stage',
-        help='Done tracks are automatically published so that they are available in frontend.')
+    is_visible_in_agenda = fields.Boolean(string='Visible in agenda',
+        help='If checked, the element will be visible in the frontend')
+    is_fully_accessible = fields.Boolean(string='Fully accessible',
```

Comment:
I suppose this is a renaming of ``is_done`` . is_done is not a good name but is_fully_accessible is strange, compared to "is_published" that we have on tracks. Maybe ``is_validated`` to indicate we have an acceptance then validation stage ?